### PR TITLE
Apply simple change

### PR DESCRIPTION
**Pull request by TaeksuKim**
Apply simple changes
- Fix a simple typo
- Exclude PyCharm configuration folder from the repository
- Code formatting
- Remove useless parenthesis
- [Replace Nested Conditional with Guard Clauses](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)

### CODE REVIEW

File: sbfl/diagnosability.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 def diversity(X):
     N, _ = X.shape
+    if N < 2:
```
Comment (agb94) [inline sbfl/diagnosability.py:5]: How about `N == 1` ? 😛

Comment (TaeksuKim) [reply sbfl/diagnosability.py:5]: Oh! I worried following case

```
>>> x = np.array([])
>>> x.shape
(0, )
```

But I found that unpacking error occurs in the case 😄 
```
>>> x = np.array([])
>>> N, M = x.shape    # Value error here
```

Thus, no problem at all for `N == 1`
Thanks a lot

Comment (TaeksuKim) [reply sbfl/diagnosability.py]: @agb94 All done :ok_hand:

**Comment by agb94** [comment]
아이고 위에껄 먼저 했더니 컨플릭이 났나봐요..

**Comment by TaeksuKim** [comment]
> 아이고 위에껄 먼저 했더니 컨플릭이 났나봐요..

@agb94 Never mind. I'll resolve the conflict.

**Comment by TaeksuKim** [comment]
Merge done :smile_cat:

**Review by agb94 — APPROVED** [review]

**Pull request merged**